backend/app/server/models/post.py

```diff
@@ -20,6 +20,7 @@ class Config:
 
 class UpdatePostModel(BaseModel):
     description: Optional[str]
+    post_id: str
```

Comment:
Python suggests optional parameters to exist after the required parameters.
(example: `def func(req_param, optional_param = None)`)
```suggestion
    post_id: str
    description: Optional[str]
```